qemu/tests/virtio_serial_file_transfer.py

```diff
@@ -218,7 +249,6 @@ def run(test, params, env):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     logging.info('Transfer data from %s' % sender)
-    result = transfer_data(params, vm, sender='both')
-    vm.destroy()
+    result = transfer_data(params, vm, sender)
```

Comment:
I will need destroy vm at the end of the test to avoid possible exceptions when running this single case. Not recommend to call a test case as background process, better use those functions`transfer_data()` in your test if you need such step.

Reply:
We have an existing driver_in_use case for all related cases, it needs a single case.
Call virtio_serial_file_transfer case directly is easier.
If you still prefer to keep vm.destroy() here, I can create another .py file. 
But it's actually dup of code except for the destroy action.

What's your opinion?

Reply:
@lijinlijin As mentioned above, please abandon the usage of `utils_test.run_virt_sub_test` for all new cases.